app/helpers/templates_helper.rb

```diff
@@ -28,6 +28,20 @@ def infobox_functions_and_macros(docs_section)
           :rel => 'external')
   end
 
+  def safemode_methods
+    Rails.cache.fetch("safemode_methods", :expires_in => 2.hours) do
+      objects = ObjectSpace.each_object(Class).select{|x| x < Safemode::Jail }
+      objects_with_methods = objects.map do |obj|
+        [obj.name.gsub(/::Jail$/, ''), obj.allowed_methods.sort.join(' ')]
+      end
+      objects_with_methods.uniq.sort{|a,b| a.first <=> b.first}
+    end
+  end
+
+  def infobox_safemode
+    alert(:class => 'alert-info', :header => '', :text => (_('In safemode only %s are allowed') % link_to_function(_('selected methods or variables'), '$("#safemode_tab").click()')))
```

Comment:
a) I'm unsure if "safemode" is the best phrase to use.  I'd suggest keeping it really simple and use "In templates", because you already know safemode is enabled.  Else I'd consider calling it "safe mode", as this is the description used in settings - the name of the gem isn't the user's business.
b) Avoid splitting the sentence into three parts over two strings, they won't be translated together and translators won't know they're from the same text. I'd suggest having one sentence, a colon, then a link that can be translated more independently (e.g. "list of available template helpers").